app/serializers/api/v1/customer_serializer.rb

```diff
@@ -16,6 +16,10 @@ class CustomerSerializer < BaseSerializer
         address(object.shipping_address)
       end
 
+      attribute :balance, if: proc { |record|
+        record.respond_to?(:balance_value)
+      }, &:balance_value
```

Comment:
Now we just check if customer responds to `balance_value`, no params passed as you suggested

Reply:
Excellent!